assets/css/_sass/website/_layout.scss

```diff
@@ -186,7 +189,7 @@ h2 {
 }
 
 p {
-  font-size: $font-size-3;
+  font-size: $base-font-size;
```

Comment:
Any reason we're switching back to the base font size of 16 px? Ty's design had the body text throughout the site at 14 px.

Reply:
I used the variables already defined to make this text 16px, rather than making a global change. The homepage tagline copy was set to 16px. See https://rackspace.invisionapp.com/d/main/#/console/11950347/252098735/preview

Reply:
The css in this location applies across the entire site. Changing this to 16px changes all of the body text. If you want to change the body text on the home page, it needs to be changed here: https://github.com/rackerlabs/design-system/blob/staging/assets/css/_sass/website/_home.scss#L20

Reply:
Reverted this change and made the change in the recommended location as mentioned above.